apps/demos/Demos/Chat/Overview/description.md

```diff
@@ -0,0 +1,27 @@
+Chat is an interactive interface that allows users to send and receive messages in real time.
+
+To get started with the DevExtreme Chat component, refer to the following step-by-step tutorial: [Getting Started with Chat](/Documentation/Guide/UI_Components/Chat/Getting_Started_with_Chat/).
+
+The demo covers Chat basics: setting initial messages, displaying new messages, managing users, and linking two chats in real-time.
+<!--split-->
+
+## Messages
+
+To set initial messages, you can either populate the [items](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#items) array (shown in this demo) or use [dataSource](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#dataSource).
+
+To render a new message, you must:
+
+- If you use **items**, update the array with the new message.
+- If using **dataSource**, implement [load](/Documentation/ApiReference/Data_Layer/CustomStore/Configuration/#load) and [insert](/Documentation/ApiReference/Data_Layer/CustomStore/Configuration/#insert) operations.
```

Comment:
I want to keep list items consistent. Don't want one to read "If you use" and another "If using". 

```suggestion
- If you use a **dataSource**, implement [load](/Documentation/ApiReference/Data_Layer/CustomStore/Configuration/#load) and [insert](/Documentation/ApiReference/Data_Layer/CustomStore/Configuration/#insert) operations.
```